src/07-even-more-loops.js

```diff
@@ -109,7 +157,15 @@ function findSmallest(array) {}
  *  findLargest([ 0, 11, -2, 5 ]);
  *  //> 11
  */
-function findLargest(array) {}
+function findLargest(array) {
```

Comment:
This is probably the best indented and formatted function in your project.  Notice how easily we can tell what code is nested where and what our operations are.  All we're missing are a few semicolons at the end of our statements.